apps/frontend/src/views/Compare.vue

```diff
@@ -260,6 +267,17 @@ export default class Compare extends Vue {
     }
   ];
 
+  compareItems = [
+    'Run Time',
+    'Execution Length',
+    'Control Count',
+    'Completed Control Count',
```

Comment:
What is the difference between `Completed` and `Passed` controls?

Reply:
This is the same as `Control Count`, will remove.